component/main.jsonnet

```
@@ -70,6 +70,11 @@ local readServices = kube.ClusterRole('appcat:services:read') + {
       resources: [ 'pods/portforward' ],
       verbs: [ 'get', 'list', 'create' ],
     },
+    {
+      apiGroups: [ '""', 'project.openshift.io' ],
```

`'""'` seems wrong/unnecessary. Why did you add that?

---

It is specified like this in the default `view` clusterRole. Not sure why though

---

Hmm interesting. Maybe some OpenShift weirdness/backwards compatibility thing?

But also to get the same output you'll need to use 

```suggestion
      apiGroups: [ '', 'project.openshift.io' ],
```